services/master/controllers/deployCheckers.go

```diff
@@ -0,0 +1,122 @@
+package controllers
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/mholt/archiver/v3"
+	checker "github.com/sdslabs/katana/services/challengecheckerservice"
+)
+
+func DeployCheckers(c *fiber.Ctx) error {
+	foldername := ""
+	fmt.Println("Starting")
+	if form, err := c.MultipartForm(); err == nil {
+
+		//sort this
+		if token := form.Value["token"]; len(token) > 0 {
+			// Get key value:
+			fmt.Println(token[0])
+			c.SendString("Test a")
```

Comment:
This code feels incomplete, finish it

Reply:
I referred Ashpect's branch here... I will see the changes he made here and apply them.

Reply:
Rebase first, I've merged Percy's PR